src/content/docs/release-notes/agent-release-notes/net-release-notes/net-agent-10-18-0.mdx

```diff
@@ -0,0 +1,42 @@
+---
+subject: .NET agent
+releaseDate: '2023-10-17'
+version: 10.18.0
+downloadLink: 'https://download.newrelic.com/dot_net_agent/latest_release'
+features: ['Enable configuration of High Security Mode via environment variable (NEW_RELIC_HIGH_SECURITY).','Enable startup logging to Event Log for all applications on Windows.','Enable startup logging to EventLog for .NET 6+ applications on Windows.']
```

Comment:
Need to remove `,'Enable startup logging to EventLog for .NET 6+ applications on Windows.'` on this line